app/views/config_groups/index.html.erb

```diff
@@ -1,7 +1,8 @@
 <% title _('Config Groups') %>
 
 <% title_actions new_link(_('Create Config Group')), help_button %>
-
+<%= alert :header => '', :class => 'alert-warning', :close => false,
+          :text => _("Puppet ENC have been extracted to a plugin and will be removed from core in 3.0") %>
```

Comment:
How about modifying the message to refer to the specific resource in each page? e.g., this will become: 
```suggestion
          :text => _("Managing config groups is being extracted to the foreman_puppet_enc plugin and will be removed from core in 3.0") %>
```